src/pass.h

```diff
@@ -429,6 +424,11 @@ class Pass {
 
   std::string name;
 
+  PassRunner* getPassRunner() { return runner; }
+  void setPassRunner(PassRunner* runner_) { runner = runner_; }
```

Comment:
Can maybe assert here that the old value was `nullptr`.

Reply:
There's a pattern where a function-parallel pass is run on the functions and then run on module code, and both code paths set the runner, so just checking for nullptr won't work. I can check for null or that the old runner is the same as the new runner, though.